CMSIS/DoxyGen/Core/src/Template.txt

```diff
@@ -377,6 +378,73 @@ The files for other compiler vendors differ slightly in the syntax, but not in t
 \verbinclude "Source/ARM/startup_Device_ac6.S"
 */
 
+/*=======0=========1=========2=========3=========4=========5=========6=========7=========8=========9=========0=========1====*/
+/**
+\page linker_sct_pg Scatter-Loading description file <device>_ac<5|6>.sct
+
+If C-Startup is used then it is necessary that also a scatter file for linking is used.
+
+The \ref linker_sct_pg contains regions for:
+ - Code (Read only data, Execute only data)
+ - RAM (Read/Write data, Zero initialized data)
+ - Stack
+ - Heap
+ - Stack Seal (Armv8-M/v8.1-M)
+ - CMSE veneer (Armv8-M/v8.1-M)
+
+Within the scatter file the user needs to specify a set of macros
```

Comment:
Within the scatter file, the user needs to specify a set of macros. The scatter file is passed through the C preprocessor which uses these macros to calculate the start address and the size of the different regions.

Reply:
changed